Core/Core/Inbox/ComposeMessage/ViewModel/ComposeMessageViewModel.swift

```diff
@@ -187,14 +247,24 @@ class ComposeMessageViewModel: ObservableObject {
             title: String(localized: "Take photo", bundle: .core),
             accessibilityIdentifier: nil
         ) { [weak self] in
-            self?.isTakePhotoVisible = true
+            guard let self else {
+                return
+            }
+            VideoRecorder.requestPermission(cameraService: cameraPermission) { isEnabled in
+                isEnabled ? self.isTakePhotoVisible = true : viewController.value.showPermissionError(.camera)
```

Comment:
This seems to me like a misuse of the ternary operator. Could you please change it to a simple if/else for readability?
Same for the same below. (and in other places if you used it elsewhere)